docs/features.md

```diff
@@ -33,7 +33,7 @@ Feature                                | ID       | Status             | Notes
 `DEVICE_RESET`                         | `0x1802` | :x:                |
 `OOBSTATE`                             | `0x1805` | :x:                |
 `CONFIG_DEVICE_PROPS`                  | `0x1806` | :x:                |
-`CHANGE_HOST`                          | `0x1814` | :x:                |
+`CHANGE_HOST`                          | `0x1814` | :x:                | implementation in progress
```

Comment:
I think we can use one of the following emoji here: :hammer: (`:hammer:`), :wrench: (`:wrench:`).

Reply:
Works for me. :-)

Reply:
No, I meant in the emoji column, instead of `:x:` :joy:

Reply:
Oh, well, probably good enough for now.  I